state-chain/pallets/cf-broadcast/src/migrations/initialize_broadcast_timeout_storage.rs

```diff
@@ -15,20 +22,21 @@ mod old {
 
 	// Same timeout values as previously defined in `#[pallet::constant]`s
 	// and same as currently used in `node/src/chain_spec.rs`
-	pub const ETHEREUM_BROADCAST_TIMEOUT: BlockNumber = 5 * MINUTES;
-	pub const POLKADOT_BROADCAST_TIMEOUT: BlockNumber = 4 * MINUTES;
-	pub const BITCOIN_BROADCAST_TIMEOUT: BlockNumber = 90 * MINUTES;
-	pub const ARBITRUM_BROADCAST_TIMEOUT: BlockNumber = 2 * MINUTES;
-	pub const SOLANA_BROADCAST_TIMEOUT: BlockNumber = 4 * MINUTES;
+	pub const ETHEREUM_BROADCAST_TIMEOUT: BlockNumber = 5 * BLOCKS_PER_MINUTE_ETHEREUM;
```

Comment:
FWIW due to rounding this works out as more like 4.5 minutes. I don't think it matters but worth keeping in mind in case we try to coordinate this with anything external.

Reply:
Added comment noting this.